sources/packages/backend/libs/services/src/workflow/workflow-client.service.ts

```diff
@@ -0,0 +1,170 @@
+import { Injectable, Logger } from "@nestjs/common";
+import { ApplicationExceptionStatus, ProgramInfoStatus } from "@sims/sims-db";
+import { CreateProcessInstanceResponse, Duration, ZBClient } from "zeebe-node";
+import {
+  APPLICATION_EXCEPTION_STATUS,
+  PROGRAM_INFO_STATUS,
+} from "./variables/assessment-gateway";
+
+@Injectable()
+export class WorkflowClientService {
+  private readonly logger = new Logger(WorkflowClientService.name);
+
+  constructor(private readonly zeebeClient: ZBClient) {}
+
+  /**
+   * Starts application assessment.
+   * @param workflowName workflow to be started.
+   * @param assessmentId student assessment that need to be processed.
+   * @returns result of the application start.
+   */
+  async startApplicationAssessment(
+    workflowName: string,
+    assessmentId: number,
+  ): Promise<CreateProcessInstanceResponse> {
+    try {
+      return await this.zeebeClient.createProcessInstance(workflowName, {
```

Comment:
mm. do we `await`?

Reply:
Yes because of the try-catch. Please keep in mind redundant `await` will be capture by Sonarcloud 😉